grammar.js

```diff
@@ -2,6 +2,7 @@ const C = require("tree-sitter-c/grammar")
 
 const PREC = Object.assign(C.PREC, {
   LAMBDA: 18,
+  USER_DEFINED_LITERAL: C.PREC.SUBSCRIPT + 1,
```

Comment:
Are UDLs specifically related to subscripts? This can probably just be '19' instead of being based on another unrelated precedence.

Reply:
No, in fact it doesn't need to be specified at all. I'll remove it.